src/middlewares/auth.validation.js

```diff
@@ -0,0 +1,52 @@
+import Response from '../services/response';
+import UserService from '../services/login.services';
+import AuthHandler from '../services/jwt';
+
+const authValidation = {
+  userExist: async (req, res, next) => {
+    const user = await UserService.getByEmail(req.body);
+    if (!user) {
+      Response.setError(401, 'You don\'t have an account. Please create an account');
```

Comment:
Just a simple correction `don't` Not `don\'t`

Reply:
Thanks. but this is for escaping the quotes so that it can be displayed